package/MDAnalysis/core/universe.py

```diff
@@ -820,6 +878,28 @@ def _process_attr(self, attr):
         for funcname, meth in attr.transplants['Universe']:
             setattr(self.__class__, funcname, meth)
 
+    def _unprocess_attr(self, attr):
+        """
+        Undo all the stuff in _process_attr.
+
+        If the topology attribute is not present, nothing happens
+        (silent fail).
+        """
+        for cls in attr.target_classes:
+            self._class_bases[cls]._del_prop(attr)
+
+        # Group transplants
+        for cls in (Atom, Residue, Segment, GroupBase,
```

Comment:
This big tuple is code smell. Can you guarantee that you will never want to add another class here? If yes, then make it a proper constant. If no, then compute it based on whatever it is that they have in common.